test/models/channel_test.js

```diff
@@ -27,15 +30,64 @@ describe('Channel', function() {
       expect(d.parameters().param1.json()).to.equal(js.parameters.param1);
     });
   });
-   
+
   describe('#parameter()', function() {
     it('should return a specific ChannelParameter object', function() {
       const d = new Channel(js);
       expect(d.parameter('param1').constructor.name).to.be.equal('ChannelParameter');
       expect(d.parameter('param1').json()).to.equal(js.parameters.param1);
     });
   });
-  
+
+  describe('#hasServers()', function() {
+    it('should return a boolean indicating if the channel has a servers list', function() {
+      const d1 = new Channel(jsWithServers);
+      const d2 = new Channel(jsWithoutServers);
+      expect(d1.hasServers()).to.equal(true);
+      expect(d2.hasServers()).to.equal(false);
+    });
+  });
+
+  describe('#servers()', function() {
+    it('should return an array of String server names or an empty array', function() {
```

Comment:
I would prefer to split this tests to two separates one, first for `jsWithServers` and second for `jsWithoutServers`, but only if you want.